### DISCUSSION TITLE
Do you account for the model’s probability of being wrong?

### DISCUSSION THREAD

**Discussion by raynerz** [kyleskom/NBA-Machine-Learning-Sports-Betting#94] [General]
First of all thanks to @nkgilley for this great project, I will make sure to contribute soon enough.

For all the users of the project, when you are betting do you account for the probability of the model being wrong (~25%). 

When analyzing my bets I always account for that by using simple simple probabilities logic. 

Say the model tells me team A has 55% of winning. I assume p(system) = .75 and p(A) = .55 meaning that the probability of team A winning is 

`p(system U a) = 0.75 * 0.55 = 0.41` 

That’s rather low, so either I don’t take the bet or combine it with another one that makes the ensemble  higher than 0.66 (which is a probability I go with) 

I know Basket is a non-random event but that should cover the risk better. Do you also take a similar approach ?

**Comment by kyleskom** [comment]
I don't actually and this is an interesting approach. Maybe this can be something to build in.

**Reply by raynerz** [reply]
Indeed. Although this approach is based on theoretical random probability which might not work in non-random situations like basketball, where so many factors contribute to the result. 

Regarding your roadmap, I saw that there is a PR already for the database. What else do you have in mind for the project? 

I’m thinking Auto ML and some MLOps to retrain the model daily. But probably would start by contributing a small thing like dockerizing it. wdyt?

**Reply by kyleskom** [reply]
Roadmap would be to clean up the code better, defiantly move to a db which we have coming. Implementing player data then playing around with more / different data and train more models.

**Reply by kyleskom** [reply]
I think really once the data is cleaner and easier to work, training more / different models and playing around with different data to achieve the best result.
